Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -45,7 +45,7 @@ For sensitive apps ("Level 2"), the MASVS adds the following:
 - Step-up authentication is required to enable actions that deal with sensitive data or transactions.
 - The app informs the user of the recent activities with their account when they log in.
 
-#### Two-Factor Authentication and Step-up Authentication
+#### Two-Factor Authentication and Step-up Authentication (MSTG‑AUTH‑9 and MSTG‑AUTH‑10)
```

Comment:
below in this section search for:

- Avoid Voicemail: If a user prefers to receive a phone call, do not leave the OTP information as a voicemail

and add the missing ending full stop.